### PR TITLE
Show original manual asset price in manual balance form

### LINKED ISSUE #5491: Show original manual asset/price combination in manual balances form
## Abstract

In the manual balance form the original price/asset should be displayed for the user's latest manual price if it is different from the users displayed currency

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #5491

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/components/accounts/manual-balances/ManualBalancesForm.vue [hunk 1]
```diff
@@ -238,16 +239,22 @@ watch(v$, ({ $invalid }) => {
...
 });
 
-watch(asset, async asset => {
+watch(asset, async () => {
   const form = get(priceForm);
-  await form?.searchAssetPrice(asset);
+  nextTick(() => {
+    form?.searchAssetPrice();
+  });
 });
```
Comment (kelsos) [inline frontend/app/src/components/accounts/manual-balances/ManualBalancesForm.vue:247]: Can't we move this watcher in the form itself instead?

File: frontend/app/src/components/accounts/manual-balances/ManualBalancesPriceForm.vue [hunk 2]
```diff
@@ -80,12 +118,20 @@ watch(isCustomPrice, isCustomPrice => {
...
+  searchAssetPrice();
+});
+
+watch(asset, () => {
+  searchAssetPrice();
+});
+
 const { tc } = useI18n();
 
 defineExpose({
```
Comment (kelsos) [inline frontend/app/src/components/accounts/manual-balances/ManualBalancesPriceForm.vue:137]: do we use expose after the changes?

File: frontend/app/src/components/history/transactions/TransactionContent.vue [hunk 3]
```diff
@@ -236,7 +236,7 @@ const saveData = async (
   if ('identifier' in event) {
     return await editTransactionEvent(omit(event, 'evmChain'));
   }
-  return await addTransactionEvent(event);
+  return await addTransactionEvent(omit(event, 'evmChain'));
```
Comment (kelsos) [inline frontend/app/src/components/history/transactions/TransactionContent.vue:239]: the backend changes for this are in develop but the frontend are only in bugfixes.
😛 hopefully it won't create a merge conflict

**Review by kelsos — APPROVED** [review]
@lukicenturi I am approving since it looks good to me.

The only comment is that if we don't really use that defineExpose there we can remove it.

File: frontend/app/src/components/history/transactions/TransactionContent.vue (hunk 3 above)
Comment (lukicenturi) [reply frontend/app/src/components/history/transactions/TransactionContent.vue]: I guess I'll just cherry pick ur commit about this to develop. There are some error in develop without this commit 😅

**Pull request merged**